GNUmakefile

```diff
@@ -36,6 +36,7 @@ SH_PROGS    = afl-plot afl-cmin afl-cmin.bash afl-whatsup afl-system-config
 MANPAGES=$(foreach p, $(PROGS) $(SH_PROGS), $(p).8) afl-as.8
 ASAN_OPTIONS=detect_leaks=0
 
+ifndef AFL_NO_LTO
```

Comment:
instead of this - is it possible to do ifneq "$(shell uname)" "Android" or something similar instead? Or check on a file that exists only on Android? Otherwise we need to document this somewhere and this makes things just more messy.
the more auto magic the better.

Reply:
We re cross building here but I can try a little "assumption" here instead.